.github/workflows/release.yml

```diff
@@ -132,6 +134,13 @@ jobs:
     if: always() && (needs.publish.result == 'success')
     runs-on: ubuntu-latest
     steps:
+      - name: Generate a token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
```

Comment:
How is this different from tibdex/github-app-token@v1, which I used for the update_dependencies workflow?

Reply:
This is the official one provided by github. The other one was used before they had official one